src/parser/jobs/mch/modules/Drift.tsx

```diff
@@ -0,0 +1,95 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import _ from 'lodash'
+import React from 'react'
+
+import ActionLink from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import {CastEvent} from 'fflogs'
+import Module, {dependency} from 'parser/core/Module'
+import Downtime from 'parser/core/modules/Downtime'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+
+// Escalate severity depending on how long drill/AA were drifted for
+const DRIFT_SEVERITY_TIERS = {
+	0.5: SEVERITY.MINOR,
+	1: SEVERITY.MEDIUM,
+	2: SEVERITY.MAJOR,
+}
+
+const GCDS = [
+	ACTIONS.DRILL.id,
+	ACTIONS.AIR_ANCHOR.id,
+]
+
+const COOLDOWN_MS = {
+	[ACTIONS.DRILL.id]: ACTIONS.DRILL.cooldown * 1000,
+	[ACTIONS.AIR_ANCHOR.id]: ACTIONS.AIR_ANCHOR.cooldown * 1000,
+}
+
+export default class Drift extends Module {
+	static handle = 'drift'
+	static title = t('mch.drift.title')`GCD Drift`
+
+	@dependency private suggestions!: Suggestions
+	@dependency private downtime!: Downtime
+
+	private castHistory: CastEvent[] = []
+
+	private cumDrift = {
```

Comment:
`drifts`? `totalDrift`? c'mon, you're killing me here.
Other than this (I can't stop looking at this), I think it's good – gonna pull down a few machinist parses and verify just to make sure.

Reply:
I for one welcome and appreciate cumdrift. With the Dirty Dancing module on DNC, we just need BRD to come? to the party. I am 12.